docs/components/Table.vue

```diff
@@ -13,13 +13,23 @@
           :no-data-text="noDataText"
           :sort-by="sortBy"
           :order="order"
+          :selection-table="selectionTable"
+          :partial-select="partialSelect"
           class="component-example-table"
           @sortTable="sortTable"
+          @massSelect="massSelection($event)"
         >
           <tr
             v-for="user in filteredUsers"
             :key="user.id"
+            @click="toggleSelection(user)"
```

Comment:
I think we should take this out even though it's just a demo, I don't want to suggest that clicking a row to select is our recommended usage